instruments/AWGs.py

```diff
@@ -16,6 +16,8 @@
 from importlib import import_module
 import os
 import inspect
+from drivers.APS import APS
```

Comment:
Ok, but that is not the problem. If you look at the output you just pasted there, you see that it failed to register the 'APS' driver, but picked up all the rest.